src/native/crt.c

```diff
@@ -582,7 +586,10 @@ struct aws_byte_cursor aws_jni_byte_cursor_from_direct_byte_buffer(JNIEnv *env,
 
 struct aws_string *aws_jni_new_string_from_jstring(JNIEnv *env, jstring str) {
     struct aws_allocator *allocator = aws_jni_get_allocator();
-    return aws_string_new_from_c_str(allocator, (*env)->GetStringUTFChars(env, str, NULL));
+    const char *str_chars = (*env)->GetStringUTFChars(env, str, NULL);
```

Comment:
Need error handling.

Reply:
For what error? If the string returned is null, it will be returned that way. Nothing else in here can fail?

Reply:
Does both `aws_string_new_from_c_str` and `ReleaseStringUTFChars` accept null args?

Reply:
yes.